qemu/tests/virtio_chardev_trace.py

```diff
@@ -0,0 +1,107 @@
+import logging
+import os
+import time
+import aexpect
+from avocado.utils import process
+from virttest import error_context
+from virttest import env_process
+from virttest import data_dir
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    virtio-trace support testing:
+    1) Make FIFO per CPU in a host.
+    2) reboot guest with virtio-serial device, control path and data path per CPU.
+    3) Download trace agent and compile it.
+    4) Enable ftrace in the guest.
+    5) Run trace agent in the guest.
+    6) Open FIFO in a host.
+    7) Start to read trace data by ordering from a host
+    8) Stop to read trace data by ordering from a host
+    9) repeat 7) and 8) with different CPU
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_procs():
+        procs = []
+        for x in range(0, int(nums_cpu)):
+            pipefile = '/tmp/virtio-trace/trace-path-cpu{}.out'.format(x)
+            proc = aexpect.run_bg('cat %s' %pipefile)
+            procs.append(proc)
+        return procs
+
+    def execute_host_cmd(host_cmd, timeout=60, shell=False, ignore_status=False):
+        """
+        Execute the host_cmd on host, limited in timeout period
+
+        param host_cmd: the host_cmd to run on host
+        param timeout: the timeout for running this command
+        return: the output of the host_cmd
+        """
+        logging.info("Executing host command: %s" % host_cmd)
+        cmd_result = process.getstatusoutput(host_cmd, timeout=timeout, shell=shell,  ignore_status=ignore_status)
```

Comment:
`cmd_result` is a tuple here. I still think we do not need add this function, just call `process.***` as needed is enough.
At least, replace `cmd_result` with `status, output`.

Reply:
Hi but it log the command that are executing so it will be useful